features/step_definitions/civil_journey_steps.rb

```diff
@@ -707,11 +707,9 @@
 # rubocop:enable Lint/Debugger
 
 Given('I start the passported journey') do
-  LaaApplyForLegalAid::Application.config.x.allow_non_passported_route = false
 end
 
 Then('I complete the passported journey') do
-  LaaApplyForLegalAid::Application.config.x.allow_non_passported_route = true
```

Comment:
If the step is empty it should probably be removed completely too, along with references to it in the features

Reply:
Yep can do! Will removing it only affect Cucumber if feature references aren't removed?

Reply:
It should do! :)